app/models/membership_application.rb

```diff
@@ -41,6 +41,9 @@ class MembershipApplication < ApplicationRecord
 
   scope :open, -> { where.not(state: [:accepted, :rejected]) }
 
+  delegate :full_name, to: :user, prefix: true, allow_nil: true
+  delegate :email, to: :user, prefix: true, allow_nil: true
+
```

Comment:
@RobertCram @weedySeaDragon so we'll go with delegating `#full_name` to `User`? TBH I can't remeber why I even added the `#email` delegate to `User`. I will just delete it.